### PR TITLE
French update

### PR DESCRIPTION
**Pull request by Toshiro-Relhax**

### CODE REVIEW

File: RelhaxModpack/RelhaxModpack/Utils/Translations.cs [hunk 1]
```diff
@@ -5041,7 +5042,7 @@ public static void LoadTranslations()
             English.Add("GcDownloadStep2TabDescription", "Close the game center (application will detect closure)");
             German.Add("GcDownloadStep2TabDescription", "Schließe das Spiel (das Programm erkennt das Schließen)");
             Polish.Add("GcDownloadStep2TabDescription", "Proszę zamknąć WG Game Center (aplikacja automatycznie wykryje zamknięcie).");
-            French.Add("GcDownloadStep2TabDescription", TranslationNeeded);
+            French.Add("GcDownloadStep2TabDescription", "Fermer le game center (L'application va détecter la fermeture");
```
Comment (Willster419) [inline RelhaxModpack/RelhaxModpack/Utils/Translations.cs]: potential missed closing parenthesis? @Toshiro-Relhax

Comment (Nullmaruzero) [reply RelhaxModpack/RelhaxModpack/Utils/Translations.cs]: My humble opinion, just fix it for him ;) He dedicated his time to finish the tranlsation, which was severely outdated. It's cruel to hold it based on a single ``)`` :D

Comment (Willster419) [reply RelhaxModpack/RelhaxModpack/Utils/Translations.cs]: bruh.

i'm not holding him on a single `)`. I will gladly do it for him. If he gets to it first or I tomorrow morning, who ever :)

Comment (Nullmaruzero) [reply RelhaxModpack/RelhaxModpack/Utils/Translations.cs]: Haha, X-mas challenge! :D

**Pull request merged**